### PR TITLE
Fix weekend crossword

### PR DESCRIPTION
**Pull request by alexduf**
This was causing 500 on MAPI side during weekends.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -25,16 +25,16 @@ scmInfo := Some(ScmInfo(
...
-        <id>robertberry</id>
-        <name>Robert Berry</name>
-        <url>https://github.com/robertberry</url>
+        <id>theguardian</id>
+        <name>The Guardian</name>
+        <url>https://github.com/guardian/</url>
       </developer>
     </developers>
-  )
+
```
Comment (NathanielBennett) [inline build.sbt]: Seriously>?? :+1:

**Review by NathanielBennett — APPROVED** [review]
:+1: Yeah i figured it would be the missing weekend type. Ship it ..

File: build.sbt (hunk 1 above)
Comment (NathanielBennett) [inline build.sbt]: (i meant the "robert berry" rather than the curly brace)

Comment (alexduf) [inline build.sbt]: I wouldn't mind but we're pushing it to maven so I may as well point to the right owner :)

**Comment by alexduf** [comment]
Thanks @NathanielBennett !

**Pull request merged**